app/lib/actions/katello/capsule_content/sync.rb

```diff
@@ -31,7 +31,6 @@ def plan(capsule_content, options = {})
               end
               plan_action(ConfigureCapsule, capsule_content, environment, content_view, repository)
               sync_repos_to_capsule(capsule_content, repository_ids)
-              plan_action(RemoveOrphans, :capsule_id => capsule_content.capsule.id)
```

Comment:
@jlsherrill I'm ok with this solution, but I think we will get some users complaining, especially if they are not aware of the rake task. I only mention this because users complaining is the reason clean orphans task was added at the end of every fpc sync :)